Cargo.toml

```diff
@@ -0,0 +1,74 @@
+[package]
+name = "railwayapp"
+version = "0.0.15"
```

Comment:
I assume the version would have to be set here?

Reply:
Cargo release should set this once we release it to crates.io